static/app/views/starfish/views/spans/queries.tsx

```diff
@@ -70,3 +75,43 @@ export const getSpansTrendsQuery = (datetime: DateTimeObject, groupIDs: string[]
     ORDER BY interval asc
   `;
 };
+
+export const useErrorRateQuery = (queryString: string) => {
+  const location = useLocation();
+
+  const interval = 12;
+  const query: NewQuery = {
+    id: undefined,
+    name: 'Db module - http error rate',
+    projects: [1],
+    fields: ['http_error_rate()'],
+    query: queryString,
+    version: 1,
+    topEvents: '5',
+    dataset: DiscoverDatasets.SPANS_METRICS,
+    interval: `${interval}h`,
+    yAxis: ['http_error_rate()'],
+  };
+
+  const eventView = EventView.fromNewQueryWithLocation(query, location);
+
+  const result = useDiscoverEventsStatsQuery<{data: [number, [{count: number}]]}>({
+    eventView,
+    referrer: 'api.starfish.api-module.http-error-rate',
+    location,
+    orgSlug: 'sentry',
+    queryExtras: {
+      interval: `${interval}h`,
+      yAxis: ['http_error_rate()'],
+    },
+  });
```

Comment:
This looks right to me, but it always returns 0? If someone could double check that, it would be greate!